sdk/security/keyvault/azadmin/rbac/utils_test.go

```diff
@@ -7,24 +7,22 @@
 package rbac_test
 
 import (
-	"context"
 	"encoding/json"
 	"fmt"
 	"os"
 	"regexp"
 	"testing"
-	"time"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
-	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
-	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
+	azcred "github.com/Azure/azure-sdk-for-go/sdk/internal/test/credential"
```

Comment:
Personally I would instead rename the package level variable something like `cred` but this is style nitpicking

Reply:
Would prefer to keep the name as `azcred`. Causes less confusion since I have variables named `cred` as well